app/src/main/AndroidManifest.xml

```diff
@@ -70,6 +71,7 @@
             android:name=".SignupActivity"
             android:label="@string/title_activity_login"
             android:parentActivityName=".SplashScreenActivity"
+            android:windowSoftInputMode="adjustPan|stateAlwaysHidden"
```

Comment:
Wouldn't this resolve the problem without needing to change the layout xml? Please check.